script.js

```diff
@@ -1,18 +1,17 @@
-const navIconClose = document.getElementById('nav-icon-close');
-const navIconHamburger = document.getElementById('nav-icon-hamburger');
-const mbNavContainer = document.querySelector('.navigation-container');
-const mbNavOverlay = document.querySelector('.overlay');
-const mbNav = document.querySelector('.mb-nav');
+((doc) => {
+    const navIconClose = doc.querySelector('#nav-icon-close');
+    const navIconHamburger = doc.querySelector('#nav-icon-hamburger');
+    const mbNavContainer = doc.querySelector('.navigation-container');
+    const mbNavOverlay = doc.querySelector('.overlay');
 
-navIconHamburger.addEventListener('click', () => {
-    mbNav.style.width = "100%"
```

Comment:
This was no longer necessary since it's a child of the container you change below.

I removed this line and set the `.mb-nav` class in the css to be `width: 100%`, and it appeared to work the same way.